src/utils/request.ts

```diff
@@ -61,7 +71,7 @@ function sendRequest(url: URL, options: RequestOptions, buffer?: Buffer): Promis
       });
     });
 
-    request.on("error", (error) => {
+    request.once("error", (error) => {
```

Comment:
https://nodejs.org/api/events.html#emitteronceeventname-listener

this just guarantees we don't leave an emitter dangling